agent/hcp/telemetry/otel_exporter_test.go

```diff
@@ -111,6 +114,58 @@ func TestExport(t *testing.T) {
 	}
 }
 
+// TestExport_CustomMetrics tests that a custom metric (hcp.otel.exporter.*) is emitted
+// for exporter operations. This test cannot be run in parallel as the metrics.NewGlobal()
+// sets a shared global sink.
+func TestExport_CustomMetrics(t *testing.T) {
```

Comment:
Test for the new exporter metrics metric, which makes a call to export/shutdown/force_flush and checks that the metric is in the inmemsink.